src/game/game.py

```diff
@@ -66,7 +66,7 @@ def players_alive(self):
         return list(filter(lambda player: player.is_alive(), self._players.values()))
 
     def log(self):
-        print(f"{self.time}: {[(player.name, player.cookies, player.income, player.hp) for username, player in self._players.items()]}")
+        print(f"{self.time}: {[(player.name, player.coins, player.get_income(), player.hp) for username, player in self._players.items()]}")
```

Comment:
This is fine, but you could also implement `__repr__` for Player if you think you'll print this often. This is just a random idea; totally up to you